### DISCUSSION TITLE
Consider switching to Codeberg

### DISCUSSION THREAD

**Discussion by MagicLike** [ventoy/Ventoy#2266] [General]
- GitHub is
  - privacy-invasive
  - Microsoft-owned
  - proprietary
  - vendor-lock-in
  - Tor-hostile
- [GitHub Copilot™ attacks copyleft licensing](https://nogithub.codeberg.page)

Moving (or at least mirroring/syncing) the whole repo and all code over to [Codeberg](https://codeberg.org), a libre, free, open-source, privacy-friendly and non-profit Git platform, running the Gittea soft-fork [Forgejo](https://forgejo.org) would be great against all those listed points.

More information:
- https://codeberg.org/Recommendations/Mirror_to_Codeberg
- https://nogithub.codeberg.page

If you need any help with the transfer I am happy to assist.

**Comment by theesfeld** [comment]
I just started the migration, its pretty painless.
- Make sure you create a private access token on GitHub to allow access to repositories for migrating
- When you set up your SSH and GPG keys on Codeberg, you need to VERIFY them before they will work!
